makeflow/src/dag_node.c

```diff
@@ -47,6 +47,7 @@ struct dag_node *dag_node_create(struct dag *d, int linenum)
 	n->resource_request = CATEGORY_ALLOCATION_FIRST;
 
 	n->umbrella_spec = NULL;
+	n->umbrella_logfile = NULL;
```

Comment:
@nhazekam , good point. 
I do not think it is very meaningful to allow the user to specify an unique umbrella log file for each rule.
I will move this into the umbrella_wrapper.